include/inviwo/core/network/processornetwork.h

```diff
@@ -249,6 +250,8 @@ class IVW_CORE_API ProcessorNetwork : public Serializable,
      */
     void clear();
 
+    void accept(NetworkVisitor& visor);
```

Comment:
Should add some details here about the order. Both in terms of network topology and also the order of the visit functions ( first processor?) 
What happens with the classes deriving from multiple ones, e.g Processor is a PropertyOwner
A short description or link to the visitor pattern would also help newcomers

Reply:
generally for a visitor pattern you will always get the most the overload for the most derived type.

Reply:
The PropertyOwner would not be used in most scenarios, but wanted it there for completeness.

Reply:
Come to think of it we probably also want to state if adding/removing elements (e.g. properties) are allowed during traversal.